plugins/labhub.py

```diff
@@ -82,45 +82,49 @@ def invite_cmd(self, msg, match):
         invitee = match.group(1)
         inviter = msg.frm.nick
 
-        if invitee == 'me':
-            user = msg.frm.nick
-            response = tenv().get_template(
-                'labhub/promotions/newcomers.jinja2.md'
-            ).render(
-                username=user,
-            )
-            self.send(msg.frm, response)
-            self.TEAMS[self.GH_ORG_NAME + ' newcomers'].invite(user)
-            self.invited_users.add(user)
-            return
-
         team = 'newcomers' if match.group(2) is None else match.group(2)
+        team = team.lower()
+
+        isDeveloper = self.TEAMS[self.GH_ORG_NAME +
+                                 ' developers'].is_member(inviter)
+        isMaintainer = self.TEAMS[self.GH_ORG_NAME +
+                                  ' maintainers'].is_member(inviter)
 
         self.log.info('{} invited {} to {}'.format(inviter, invitee, team))
 
-        if self.TEAMS[self.GH_ORG_NAME + ' maintainers'].is_member(inviter):
-            valid_teams = ['newcomers', 'developers', 'maintainers']
-            if team.lower() not in valid_teams:
-                return 'Please select from one of the ' + ', '.join(valid_teams)
+        valid_teams = ['newcomers', 'developers', 'maintainers']
+        if team not in valid_teams:
+            return 'Please select from one of the ' + ', '.join(valid_teams)
+
+        def invite(invitee, team):
+
             team_mapping = {
                 'newcomers': self.GH_ORG_NAME + ' newcomers',
                 'developers': self.GH_ORG_NAME + ' developers',
                 'maintainers': self.GH_ORG_NAME + ' maintainers'
             }
 
-            # send the invite
-            self.TEAMS[team_mapping[team.lower()]].invite(invitee)
+            self.TEAMS[team_mapping[team]].invite(invitee)
+
+        if isDeveloper and team == 'newcomers':
+            invite(invitee, team)
+            return tenv().get_template(
+                'labhub/promotions/{}.jinja2.md'.format(team)
+            ).render(
+                targer=invitee,
+            )
+        elif isMaintainer:
+            invite(invitee, team)
             return tenv().get_template(
-                'labhub/promotions/{}.jinja2.md'.format(team.lower())
+                'labhub/promotions/{}.jinja2.md'.format(team)
             ).render(
                 target=invitee,
             )
         else:
             return tenv().get_template(
-                'labhub/errors/not-maintainer.jinja2.md'
+                'labhub/errors/not-eligible-invite.jinja2.md'
```

Comment:
What if developer tries to invites members to dev team and maintainers team? This error message returns newcomer can't invite members, which is not correct in here.

Reply:
Oh, I thanks @Vamshi99. I forgot to include that message  :sweat_smile: